app/views/pages/api_release_notes.html.haml

```diff
@@ -9,6 +9,88 @@
     Github issues
   for bugs or other problems.
 
+%section.api-documentation
+  %hr
+  %h2 25th February 2019
+  %ul.list-bullet
+    %li
+      Addition of Advocate supplementary claim type endpoints
+      %br/
+      %br/
+      In line with the web application interface, new endpoints to facilitate creation of "Advocate supplementary claims" have been added.
+      %br/
+      %br/
+
+      The new endpoints are interactively documented at the links below:
+      %ul.list-bullet
+        %li
+          %a{ href: '/api/documentation?v=1#!/api/postApiExternalUsersClaimsAdvocatesSupplementary', target: '_blank' } advocate supplementary claim creation
+        %li
+          %a{ href: '/api/documentation?v=1#!/api/postApiExternalUsersClaimsAdvocatesSupplementaryValidate', target: '_blank' } advocate supplementary claim validate
+      %br/
+
+      Please note that only the following miscellaneous fees are claimable via this claim type currently:
+      %br/
+      %table
+        %thead
+          %tr
+            %th
+              Unique code
+            %th
+              Description
+        %tbody
+          %tr
+            %td
+              MIWPF
+            %td
+              Wasted preparation fee
+          %tr
+            %td
+              MISPF
+            %td
+              Special preparation fee
+          %tr
+            %td
+              MIDSU
+            %td
+              Deferred sentence hearings uplift
+          %tr
+            %td
+              MIDSE
+            %td
+              Deferred sentence hearings
+          %tr
+            %td
+              MIDTW
+            %td
+              Confiscation hearings (whole day)
+          %tr
+            %td
+              MIDWU
+            %td
+              Confiscation hearings (whole day uplift)
+          %tr
+            %td
+              MIDTH
+            %td
+              Confiscation hearings (half day)
+          %tr
+            %td
+              MIDHU
+            %td
+              Confiscation hearings (half day uplift)
+
+%section.api-documentation
+  %hr
+  %h2 25th February 2019
+  %ul.list-bullet
+    %li
+      Added `unique_code` filter options to fee_types endpoint
```

Comment:
In previous posts this kind of string (referring to code) is it a `code` format - might make sense to do this here as it can be easier to read.

Reply:
👍 fixing - currently `%code.code` is needed and breaks up the haml unpleasantly. a helper (such as redcarpet for markdown would be good - but yet another dependency)